public/index.html

```diff
@@ -23,6 +23,11 @@
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
+    <!-- Workaround for Tone.js logging. -->
+    <script type="text/javascript">
+      window.TONE_SILENCE_LOGGING = true;
```

Comment:
This setting of `TONE_SILENCE_LOGGING` will take effect for the running application but does not change the tests. I still see logging in the test output, such as:

```
PASS  src/App.test.js
  ● Console

    console.log
       * Tone.js v14.7.77 *

      at Module.o (node_modules/tone/build/webpack:/Tone/Tone/core/Global.ts:72:10)
```

Reply:
The only way I could find to get rid of this was to make `AudioManagerImpl` a dynamic import.  You can see what I've done in the latest commits.  If it's too much, we'll have to live with it, as I couldn't find any other way to set that variable before Tone got a chance to fire its startup message on the console.  I tried:

1. Updating the global window object provided by Enzyme.
2. Mocking the global window object and having it reply with `true` when queried for that property.
3. Temporarily reassigning `console.log` to a dummy function.